internal/manager/config.go

```diff
@@ -184,6 +184,7 @@ func (c *Config) FlagSet() *pflag.FlagSet {
 	flagSet.BoolVar(&c.KongClusterPluginEnabled, "enable-controller-kongclusterplugin", true, "Enable the KongClusterPlugin controller.")
 	flagSet.BoolVar(&c.KongPluginEnabled, "enable-controller-kongplugin", true, "Enable the KongPlugin controller.")
 	flagSet.BoolVar(&c.KongConsumerEnabled, "enable-controller-kongconsumer", true, "Enable the KongConsumer controller. ")
+	flagSet.BoolVar(&c.IngressClassParametersEnabled, "enable-controller-ingress-class-params", false, "Enable or disable the IngressClassParameters controller.")
```

Comment:
This goes again our previous commitment to releasing this functionality behind a feature gate to get started?

Reply:
Yes it does. I'll put all this functionality ( to the extent it's possible ) behind a feature gate later on but thanks for pointing this out so that we don't miss it.

The question I had in the KEP, which I believe was left unanswered is how do we want to approach the CRD and the feature gate situation? Do we add the CRD support regardless ( e.g. to the manifests etc) but react on it when feature gate is enabled?

Reply:
ping on this one.

Reply:
> Do we add the CRD support regardless ( e.g. to the manifests etc) but react on it when feature gate is enabled?

What we did with `Gateway` support is we didn't add anything by default, and the users are responsible for turning on the gate and then adding CRDs and resource updates according to a guide. I would suggest we follow that model here unless we have good reason to deviate.

Reply:
Converted the runtime flag to a feature flag. PTAL